scripts/RuCl-GSEE-RE.py

```diff
@@ -0,0 +1,294 @@
+import os
```

Comment:
Looking at it now, a lot of the things here are just copied and pasted from the previous script. It maybe best to use the previous script, RuCl-RE.py and through flag usage, modify how it should perform. Maintaining two scripts where majority of the work is the same except for the algorithm usage is too hectic.